doc/user/tutorials/video-series-overview.md

```diff
@@ -1,56 +1,45 @@
 # Beginner Video Series
 
 ## Overview
-In this video series, you will become familiar with QGIS, download and browse the QGreenland Core data package, and explore the basic geospatial data anlysis tools that make QGreenland a powerful tool for research, education, and local decision making. The video tutorial series contains 7 short video sessions.
+In this video series, you will become familiar with QGIS, learn how to download and browse the QGreenland Core data package, and explore the basic geospatial data anlysis tools that make QGreenland a powerful tool for research, education, and local decision making. The video tutorial series contains 7 short video sessions.
 
-```
-### Goals
+```{goals}
```

Comment:
RTD complains about this line:

```
/home/docs/checkouts/readthedocs.org/user_builds/qgreenland/checkouts/722/doc/user/tutorials/video-series-overview.md:6: ERROR: Unknown directive type "goals".
```

This syntax looks like you want to do an "admonition". `goal` isn't a valid one; only the following choices are available:

`attention`, `caution`, `danger`, `error`, `hint`, `important`, `note`, `tip`, `warning`

You can create one with a custom title like: 

```
:::{admonition} My custom title with *Markdown*!
:class: tip

This is tip-style admonition with a custom title.
:::
```

Here the `class` line sets the style of the admonition to be the same as a `tip`.

https://myst-parser.readthedocs.io/en/latest/syntax/admonitions.html

Reply:
THANK YOU! I'll fix that now